checker/manual/troubleshooting.tex

```diff
@@ -741,10 +736,30 @@ \subsection{Build the Checker Framework\label{building}}
 \end{Verbatim}
 % $ to unconfuse Emacs LaTeX mode
 
-\item Add \code{tools.jar} and \code{checker.jar} to your classpath.
-  (If you do not do this, you will have to supply the \code{-cp} option
-  whenever you run \code{javac} and use a checker plugin.)
-  Example command:
+Once \<checker.jar> is built, you may wish to put the Checker Framework \<javac> even earlier on
+your path. The Checker Framework's \code{javac} ensures that \<checker.jar> 
+is on your classpath and bootclasspath, but is otherwise
+identical to the Type Annotations compiler:
+
+\begin{Verbatim}
+export PATH=$JSR308/checker-framework/checker/bin:$JSR308/jsr308-langtools/dist/bin:${PATH}
+\end{Verbatim}
+
+If you are developing a checker and wish to use the latest builds of your source files compiled by
+an IDE (such as Eclipse), you can use the development version of javac instead. This version will use compiled
```

Comment:
I think calling it "the development version of javac" makes it sound too different and scary.  It's exactly the same javac, just with different command-line options.